lib/routing/router.dart

```diff
@@ -12,20 +12,20 @@ import 'package:soundspace/routing/routes.dart';
 
 Route<dynamic> generateRoute(RouteSettings settings) {
   switch (settings.name) {
-    case HomePageRoute:
-      return _getPageRoute(HomePage());
-    case TrendingPageRoute:
-      return _getPageRoute(TrendingPage());
-    case UploadPageRoute:
-      return _getPageRoute(UploadPage());
-    case MusicPageRoute:
-      return _getPageRoute(TmpMusicPage()); // TMP MUSIC PAGE BEING USED
-    case AccountPageRoute:
-      return _getPageRoute(AccountPage());
-    case ApprovalPageRoute:
-      return _getPageRoute(ApprovalPage());
+    case homePageRoute:
+      return _getPageRoute(const HomePage());
+    case trendingPageRoute:
+      return _getPageRoute(const TrendingPage());
+    case uploadPageRoute:
+      return _getPageRoute(const UploadPage());
+    case musicPageRoute:
+      return _getPageRoute(const TmpMusicPage()); // TMP MUSIC PAGE BEING USED
```

Comment:
tmp music is stateful so Im not sure this will hold

Reply:
i dont think it works that way. It might be useful to think of it as passing a value by const reference in C++